run.py

```diff
@@ -3,20 +3,25 @@
 import os
 
 
-path_to_runner = "./Debug/SCP.exe"
+path_to_runner = "./x64/Debug/SCP.exe"
```

Comment:
okay - I will leave a #TODO note for now, and will try and get that added later today